src/frontend/web_application/.eslintrc

```diff
@@ -1,19 +1,10 @@
 {
   "extends": "airbnb",
   "parser": "babel-eslint",
-  "ecmaFeatures": {
-    "modules": true
-  },
-  "parserOptions": {
-    "ecmaVersion": 6,
-    "ecmaFeatures": {
-      "experimentalObjectRestSpread": true
-    }
-  },
+  "root": true,
   "env": {
     "browser": true,
-    "es6": true,
-    "jasmine": true,
```

Comment:
Humm you’re right their some errors on Jenkins about it, weird because I checked previously.